Gopkg.lock

```diff
@@ -245,32 +245,31 @@
     "blake2b",
   ]
   pruneopts = "UT"
-  revision = "613d6eafa307c6881a737a3c35c0e312e8d3a8c5"
+  revision = "614d502a4dac94afa3a6ce146bd1736da82514c6"
 
 [[projects]]
   branch = "master"
-  digest = "1:71200e12202b1c5bf404b0ae7f9ace57d083ebb89197f709ccc2e02295b0b8c8"
+  digest = "1:c822b34344031b25ffa32de9384a5059abd019171d4a131437daf776bd0a3b32"
   name = "golang.org/x/net"
   packages = [
     "http/httpguts",
     "http2",
     "http2/hpack",
     "idna",
-    "lex/httplex",
   ]
   pruneopts = "UT"
-  revision = "5f9ae10d9af5b1c89ae6904293b14b064d4ada23"
+  revision = "922f4815f713f213882e8ef45e0d315b164d705c"
 
 [[projects]]
   branch = "master"
-  digest = "1:c78454799563137757c640be87d6cf532e8c31015ac21eb441c694fffeadf277"
+  digest = "1:a0e12bc26f317c0e2d497baf767285e1790e526e8dd46553c5a67fcbc8692157"
   name = "golang.org/x/sys"
   packages = [
     "cpu",
     "unix",
   ]
   pruneopts = "UT"
-  revision = "78d5f264b493f125018180c204871ecf58a2dce1"
+  revision = "3b58ed4ad3395d483fc92d5d14123ce2c3581fec"
```

Comment:
I don't think that file is necessary either. Do you need to update it ?

Reply:
Nope. 
When I `dep ensure` it changed automatically.